install/deploy_all_components.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+JSONPATH1="{.spec.replicas}"
+JSONPATH2="{.status.readyReplicas}"
+
+helm install ../charts/zookeeper-operator --name zoo-op
+kubectl rollout status deploy/zoo-op-zookeeper-operator
+
+helm install ../charts/zookeeper --name zoo
+
+replicas=$(kubectl get zookeepercluster zoo-zookeeper -o jsonpath=$JSONPATH1)
+ready=$(kubectl get zookeepercluster zoo-zookeeper -o jsonpath=$JSONPATH2)
+
+until [ "$replicas" == "$ready" ]
+do
+	sleep 1;
+	replicas=$(kubectl get zookeepercluster zoo-zookeeper -o jsonpath=$JSONPATH1)
+	ready=$(kubectl get zookeepercluster zoo-zookeeper -o jsonpath=$JSONPATH2)
+done
+
+helm install ../charts/bookkeeper-operator --name pr
+kubectl rollout status deploy/pr-bookkeeper-operator
+
+helm install ../charts/bookkeeper --name br --set zookeeperUri=zoo-zookeeper-client:2181
```

Comment:
Value for zookeeper-uri should not be hard-coded, it should be picked from zookeeper chart.